.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- **NOTE: I will add the changes you need to implement using these inline comments. Please check carefully where you need to implement the changes below:**

**README file**

- Hey, great adding the readme file, however, you need to personalize it to suit your project needs to make it look professional. kindly do the following:

- change the `Project Name` to `Hello Microverse`
- Add a little description of the project. like the aim of the project
- Remove the `Live Demo (if available)` for now, except you know how to use the `gh` pages to add live link. You will learn this later in the curriculum
- In the `Builth With` header, Add this:
```

- HTML
- CSS
```
- In the `Getting Started` header, add the following:
```

To get a local copy up and running:

1. Clone this repository or download the zip folder:

**``git clone https://github.com/Mehria-Rahmani/Hello-Microverse.git``**

1. Navigate to the location of the folder in your machine:

**``you@your-Pc-name:~$ cd <folder>``**

```
- In the `Authors` header, change it to be singular, and the add replace the links to your personal detail in the social media networks...  Here:

```
## Author

👤 **your name**

- Github: [@your nme](your GitHub link)
- Twitter: [@your name](your Twitter link) # if and only if you have one
- LinkedIn: [@your name](your Linkedin link) # if and only if you have one
```
- In the `Contribution header` change the issue link to match the issue link of your project like this 
```
Feel free to check the [issue page](https://github.com/HtetNaing0814/Hello-World/issues)
```

- Update the `Acknowledgement` header by adding the microverse official link:
```
 [Microverse](https://bit.ly/MicroverseTN)
```
- In the `License` header add the content in the `MIT.md` file. 
You either add it as a file or add it as a link to the license, like this:
```
This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md
) licensed # as a link

This project is [MIT](./MIT.md) licensed # add a file by adding the `MIT.md` file in the root folder of your `Hello Microverse Project` 

```

Reply:
- [ ] **[REQUIRED CHANGES]**

**Pull Request**

- Let's update our PR by adding a summary describing introduced changes in more detail to make our PR more descriptive. we can try something like:

```
**In this project, I did the following;**

- Added an h1 in the `index.html` file
- Styled the h1 in the `style.css` file
- Ignored the `test.md` and `node_modules` file 
- Ensured there was no linter error
```

Reply:
**[REQUIRED CHANGES]**

- Kindly add some content in the `index.html` file to meet the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and fix the linter error.

Reply:
**[REQUIRED CHANGES]**

- Kindly add some content in the `styles.css` file to meet the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) and fix the linter error.